src/writing-your-own-application/index.md

```diff
@@ -2,11 +2,14 @@
 
 With the appropriate Rust compiler and toolchain installed, you're now ready to create an application.
 
-There are essentially two ways to do this: generating from a template or starting from scratch using only `cargo`.
+You can write an application in the following ways:
 
-We strongly recomend starting projects from templates as it gives you a configured project saving all the time that would require
-to setup a project started from scratch with `cargo`.
+- Generate from a template: (**Strongly recommended**). Gives you a configured project, saves time and prevents possible errors.
+- Start from scratch using Cargo: Requires more expertise since you need to configure several parts of the project.
```

Comment:
Starting from scratch with Cargo, does not give more flexibility, you can have the same flexibility using the template, and probably, with less work!

IMHO, there is no point in using Cargo to start a project, but its the standard way in Rust, that why its worth mentioning it

Reply:
Then it might be good to say that the second way doesn't provide advantages... and is mentioned here just because this is a standard way.

Feel free to disregard it.